ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -485,14 +526,28 @@ private ReplicaMetadataResponse getReplicaMetadataResponse(List<RemoteReplicaInf
           remoteReplicaInfo.getReplicaId(), messageInfo.getStoreKey());
     }
 
-    Set<StoreKey> missingStoreKeys = remoteReplicaInfo.getLocalStore().findMissingKeys(storeKeysToCheck);
-    for (StoreKey storeKey : missingStoreKeys) {
+    Map<StoreKey, StoreKey> unconvertedToConverted = new HashMap<>();
```

Comment:
I prefer the term `original`. Unconverted sort of sounds like part of a set was converted and the rest weren't. Would like the name to convey that we are dealing with two "classes" of keys.

Reply:
ok